src/vncserver@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Remote desktop service (VNC)
+After=syslog.target network.target
+
+[Service]
+Type=simple
+User={{ vnc_username }}
+PIDFile=/home/{{ vnc_username }}/.vnc/%H%i.pid
+ExecStartPre=/bin/sh -c "/usr/bin/vncserver -kill :%i > /dev/null 2>&1 || :"
+ExecStart=/usr/bin/vncserver :%i -localhost no -alwaysshared -fg -geometry 1920x1080 -SecurityTypes VncAuth
```

Comment:
Is there an already-existing vncserver unit file that you could modify in situ?  That would be less fragile.

Reply:
Yes that `vncserver@.service` unit file is installed by `ansible-role-vnc-server`.  I figured it was cleaner to simply drop in my own unit file so that I have more control over it.

I could change this so that I use ansible's `lineinfile` to append `-SecurityTypes VncAuth` to the end of the `ExecStart` line.  One issue with that is if there's already a `-SecurityTypes` parameter on that line - any thoughts on how to handle that?

Reply:
The issue with the current way of doing things is that if the unit file installed by `ansible-role-vnc-server` changes then you won't get those changes.

[replace](https://docs.ansible.com/ansible/latest/modules/replace_module.html#replace-module) might be a better fit than lineinfile for this.

Reply:
Thanks for the suggestion @jsf9k - I did just that in 4b17278